c_api/tests/c_api_behavior_test.cpp

```diff
@@ -23,6 +23,8 @@ TEST_F(CapiTest, TestBehaviorCreateRuntime) {
     TiRuntime runtime = ti_create_runtime(arch);
     TI_ASSERT(runtime == TI_NULL_HANDLE);
     CHECK_TAICHI_ERROR_IS(TI_ERROR_NOT_SUPPORTED);
+    ti_destroy_runtime(runtime);
```

Comment:
well you dont have to destroy if its already failing... it is an expected behavior to recover everything when a create call fails.